worker/solution.go

```diff
@@ -34,7 +34,7 @@ func getDataForSolutionRunner(taskId, userId, submissionNumber int64) (TaskForRu
 	var task TaskForRunner
 
 	// Get the tar.gz file from the storage
-	requestUrl := fmt.Sprintf("http://host.docker.internal:8080/getSolutionPackage?taskID=%d&userID=%d&submissionNumber=%d", taskId, userId, submissionNumber)
+	requestUrl := fmt.Sprintf("http://host.docker.internal:8888/getSolutionPackage?taskID=%d&userID=%d&submissionNumber=%d", taskId, userId, submissionNumber)
```

Comment:
what the 🐓, I think we should pass address of filestorage from config and env vars. Also you can use aliases, such as container name